source/funkin/backend/scripting/HScript.hx

```diff
@@ -38,6 +38,10 @@ class HScript extends Script {
 		interp.importFailedCallback = importFailedCallback;
 		interp.staticVariables = Script.staticVariables;
 		interp.allowStaticVariables = interp.allowPublicVariables = true;
+		if(Options.safemode)
+		{
+			interp.importBlocklist = Script.getBlacklistedImports();
```

Comment:
Btw

`__script__.interp.importBlocklist` is accessable in a script

Reply:
well, there is the problem. I see it's not possible to deny the access to "importBlocklist" without breaking the entire engine.